CONTRIBUTING.md

```diff
@@ -102,6 +102,9 @@ Meaning we'd be happy for anyone to do these tasks.**
 If you want to work on something, assign yourself to it or write a comment that
 you want to work on it and what you plan to do.
 
+- There's an [introduction for developers](docs/docs/guides/intro_developers.md)
```

Comment:
```suggestion
- There's an [introduction for developers](docs/docs/guides/developers.md)
```

Reply:
i'd suggest just using /docs/docs/guides/developers.md for now so can just have one single "Developers" guide to keep it simple and also rename the file itself.

Reply:
Done.